conda/recipes/libcudf_kafka/meta.yaml

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2018, NVIDIA CORPORATION.
+
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
+{% set minor_version =  version.split('.')[0] + '.' + version.split('.')[1] %}
+{% set cuda_version='.'.join(environ.get('CUDA_VERSION', '10.0').split('.')[:2]) %}
+
+package:
+  name: libcudf_kafka
+  version: {{ version }}
+
+source:
+  git_url: ../../..
+
+build:
+  number: {{ GIT_DESCRIBE_NUMBER }}
+  string: cuda{{ cuda_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  script_env:
+    - CC
+    - CXX
+    - CUDAHOSTCXX
+    - PARALLEL_LEVEL
+    - VERSION_SUFFIX
+  run_exports:
+    - {{ pin_subpackage("libcudf_kafka", max_pin="x.x") }}
+
+requirements:
+  build:
+    - cmake >=3.14.0
+  host:
+    - cudf {{ version }}
+    - librdkafka=1.4.0
+    - python-confluent-kafka
+  run:
+    - {{ pin_compatible('cudatoolkit', max_pin='x.x') }}
+    - librdkafka=1.4.0
+    - python-confluent-kafka
```

Comment:
Need to either split this recipe into two separate recipes for `libcudf_kafka` and `cudf_kafka` or turn this into a split output recipe. Either way, `libcudf_kafka` shouldn't depend on the `cudf` python library, just `libcudf`, and I would imagine similarly for `python-confluent-kafka`?

Reply:
Ok, I split it out but haven't made the commit with that change yet. When I make it I will let you determine if that is what you had in mind or not and resolve this if it is